### DISCUSSION TITLE
[Meetup] Głosowanie na sesje na 43 spotkanie Microsoft Azure User Group Poland

### DISCUSSION THREAD

**Discussion by kaluzaaa** [AzureCommunityPL/group#76] [Wydarzenia]
Przygotowania do 43 meetupu trwają‼️
Chcesz mieć wpływ to zagłosuj na zgłoszone sesje oraz o której ma się zacząć meetup oraz ile trwać 😎

Głosowanie do 30.09 do 23:00‼️

## [Tutaj głosujemy](https://bit.ly/32NVlIZ)

**Comment by mgrabarz** [comment]
Podbijam temat, zapraszamy do oddawania głosów. :)

**Comment by GetoXs** [comment]
Wbiłem tutaj tylko po to aby napisać: Super pomysł z głosowaniem! 😀🗳

**Reply by kaluzaaa** [reply]
Dzięki! :)

**Comment by kaluzaaa** [comment]
No to mamy wyniki głosowania na następny meetup. Zdecydowania wygrały 2h długości i start o 18:00.

Co do agendy to dwie pierwsze sesje to różnica jednego głosu‼️

Agenda to:
- Gość specjalny Thomasa Mauera(Senior Cloud Advocate at Microsoft) z Manage and govern your hybrid servers using Azure Arc
- Multi-stage pipelines in Azure DevOps - Patryk Lotzwi

![image](https://user-images.githubusercontent.com/4700951/94796284-db315b00-03de-11eb-9677-225d82a99a5b.png)
![image](https://user-images.githubusercontent.com/4700951/94796299-df5d7880-03de-11eb-9994-142d4e1a9d74.png)

**Reply by kamil-mrzyglod** [reply]
🔥🔥🔥
